configure.ac

```diff
@@ -1528,15 +1528,21 @@ struct sockaddr_storage {
 #endif
 
 #ifdef WIN32
-#define SASL_ROOT_KEY "SOFTWARE\\Carnegie Mellon\\Project Cyrus\\SASL Library"
-#define SASL_PLUGIN_PATH_ATTR "SearchPath"
-#define SASL_CONF_PATH_ATTR "ConfFile"
+#define SASL_ROOT_KEY "SOFTWARE\\Project Cyrus\\SASL Library"
```

Comment:
Would upgrade instructions be in order? I notice this path wasn't documented anywhere else, should it?

Reply:
Probably a good idea. Dunno which doc section that would go in.

Reply:
Done